digits/config/tensorflow.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+import os
+import platform
+from subprocess import Popen, PIPE
+
+from . import option_list
+
+VARNAME_ENV_TFPY = 'TENSORFLOW_PYTHON'
+DEFAULT_PYTHON_EXE = 'python2'  # @TODO(tzaman) - use the python executable that was used to launch digits?
+
+if platform.system() == 'Darwin':
+    # DYLD_LIBRARY_PATH and LD_LIBRARY_PATH is sometimes stripped, and the cuda libraries might need it
+    if "DYLD_LIBRARY_PATH" not in os.environ:
+        if "CUDA_HOME" in os.environ:
+            os.environ["DYLD_LIBRARY_PATH"] = str(os.environ["CUDA_HOME"] + '/lib')
```

Comment:
Why are we fixing people's bad environment for them here? Shouldn't this be a standard TF configuration issue?